packages/frontend/src/components/UserSettings/UserAttributesPanel/index.tsx

```diff
@@ -0,0 +1,243 @@
+    const [isDeleteDialogOpen, setIsDeleteDialogOpen] = useState(false);
+    const [isEditDialogOpen, setIsEditDialogOpen] = useState(false);
+
+    const { mutate: deleteUserAttribute } = useUserAttributesDeleteMutation();
+
+    return (
+        <>
+            <tr>
+                <td width={800}>
+                    <Flex justify="space-between" align="center">
+                        <Stack spacing="xs">
+                            <Group spacing="two">
+                                <Text>{orgUserAttribute.name}</Text>
+                                {orgUserAttribute.description && (
+                                    <Tooltip
+                                        multiline
+                                        maw={300}
+                                        withArrow
+                                        label={orgUserAttribute.description}
+                                    >
+                                        <MantineIcon
+                                            icon={IconInfoCircle}
+                                            color="gray.6"
+                                        />
+                                    </Tooltip>
+                                )}
+                            </Group>
+                            <Text fz="xs" color="gray.6">
+                                {' '}
+                                {orgUserAttribute.users.length} user
+                                {orgUserAttribute.users.length != 1 ? 's' : ''}
+                            </Text>
+                        </Stack>
+                    </Flex>
+                </td>
+                <td>
+                    <Flex justify="space-between" align="right">
+                        <Button
+                            variant="subtle"
+                            onClick={() => setIsEditDialogOpen(true)}
+                        >
+                            <MantineIcon
+                                icon={IconEdit}
+                                size="xlg"
+                                color="gray.7"
+                            />
+                        </Button>
+                        {isEditDialogOpen && (
+                            <UserAttributeModal
+                                opened={isEditDialogOpen}
+                                userAttribute={orgUserAttribute}
+                                onClose={() => setIsEditDialogOpen(false)}
+                                allUserAttributes={allUserAttributes}
+                            />
+                        )}
+                        <Button
+                            leftIcon={<MantineIcon icon={IconCircleX} />}
+                            variant="outline"
+                            onClick={() => setIsDeleteDialogOpen(true)}
+                            color="red"
+                        >
+                            Delete
+                        </Button>
+                        <Modal
```

Comment:
ideally we can render model once and keep in state uuid of user attribute that is open

Reply:
also, this will help the readability to define this modal as a separate component or even in separate file